File: Group 57/level 30/Homework/hw6.py
# 6) მომხმარებელს შემოატანინეთ 5 რიცხვი და დაამატეთ ეს რიცხვები სიაში, ბოლოს გამოიტანეთ ტერმინალში ეს სია.

list = []

i = 0

while i < 5:
    num = int(input("Enter Your Number: "))
    list.append(num)
    i += 1

print(list)      

File: Group 57/level 143/Homework/hw6.js
// 6)მომხმარებელს შემოატანინე რიცხვი.
// switch(true) გამოყენებით განსაზღვრე:

// თუ არის 0

// თუ არის ლუწი

// თუ არის კენტი

const num = parseInt(prompt("Enter a number:"));

switch (true) {
    case num === 0:
        console.log("Number is 0");
        break;
    case num % 2 === 0:
        console.log("Number is even");
        break;
    case num % 2 !== 0:
        console.log("Number is odd");
        break;
    default:
        console.log("Invalid input");
}